packages/regl-worldview/src/stories/GLText.stories.js

```diff
@@ -115,22 +160,60 @@ storiesOf("Worldview/GLText", module)
       });
 
       return (
-        <>
-          <div style={{ width: "100%", height: "95%" }}>
+        <div style={{ width: "100%", height: "100%" }}>
+          <div style={{ width: "100%", height: "100%" }}>
             <Container cameraState={{ perspective: true, distance: 40 }} backgroundColor={[0.2, 0.2, 0.4, 1]}>
               <GLText autoBackgroundColor>{markers}</GLText>
               <Axes />
             </Container>
           </div>
+          <div style={{ position: "absolute", top: "10px", right: "10px" }}>
+            <label htmlFor="search">Search: </label>
+            <input
+              type="text"
+              name="search"
+              placeholder="search text"
+              value={searchText}
+              onChange={(e) => setSearchText(e.target.value)}
+            />
+          </div>
+        </div>
+      );
+    };
 
-          <input
-            style={{ width: "100%" }}
-            type="text"
-            placeholder="search text"
-            value={searchText}
-            onChange={(e) => setSearchText(e.target.value)}
-          />
-        </>
+    return <Example />;
+  })
+  .add("custom highlight color", () => {
+    const Example = () => {
+      const [highlightColor, setHighlightColor] = useState<Color>({ r: 1, b: 0.5, g: 0.5, a: 1 });
+      const markers = textMarkers({ text: "Hello\nWorldview" }).map((marker) => ({
+        ...marker,
+        highlightedIndices: [0, 1, 2, 3, 4],
```

Comment:
As said in person, I'd change this list a bit to make sure we actually test this properly.